templates/pin.pref.epp

```diff
@@ -1,7 +1,7 @@
 <%- | $name, $pin_release, $release, $codename, $release_version, $component, $originator, $label, $version, $origin, $explanation, $packages_string, $priority | -%>
 <%-
 $pin =
-if $pin_release.length > 0 {
+if $pin_release {
```

Comment:
would this be better
`if $pin_release and $pin_release.length > 0`
Following the logic of the rest of the template.
Many thanks for the code !!! @paladox

Reply:
Thanks, done.